tests/test_cli_socket_autodiscovery.py

```
@@ -610,6 +832,270 @@ def test_base_debug_cli_discovers_cmux_tag(cli_path: str) -> bool:
+            "cmux",
+            "com.cmuxterm.app",
+        )
+        proc = run_ping(stable_cli, home)
+
+        squatter_server.join(timeout=2.0)
+        fallback_server.join(timeout=2.0)
+        for path in [default_socket, fallback_socket]:
+            try:
+                os.remove(path)
+            except OSError:
+                pass
```

Ruff (0.15.13)

[warning] Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(OSError): ...`

(SIM105)